### ISSUE TITLE
[Adaptive Cards- Web Render Test on Teams]: Voice Over does not announce the descriptive information while navigating on the 22%(2) under the polls.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#407]
### Target Platforms

iOS

### SDK Version

7.13

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 18.5
Teams Application Version:  7.13
Screen Reader: Voice Over

**Pre-Requisite:**
Turn on Voice Over.

**Repro Steps:**
1. Open the application using double tap.
2. Perform right swipe, navigate to the AC Web render group, and double tap to enter in the group.
3. Navigate to any poll using swipe gesture.
4. Now perform right swipe and navigate to the total number of response and observe the issue.

**Actual Result:**
While navigating on the total number of responses along with number of people who respond on the poll, voice over does not convey the descriptive information. Here is announcing only as **'22 percentage left paren 2 right paren.'**

**Expected Result:**
Screen Reader should announce the descriptive information while navigating on the total number of responses and here it should announce something like **"22 percent with number of 2 responses."**

**Note:**
Same issue is repro throughout the polls.

**User Impact:**
It will impact the screen reader user as they will not be able to know the descriptive information about the response that how many peoples response on the poll.

**WCAG Reference:**
[Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) 

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/3cf7768a-1fa6-412e-8326-174196a0cde8

<img width="1079" height="676" alt="Image" src="https://github.com/user-attachments/assets/6d6f4da6-0d1d-47ee-bcf5-bcbc7aeb452c" />

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue as of now this issue is moved in Teams ADO.
